src/components/pages/search.js

```diff
@@ -35,14 +37,26 @@ function Search({ initialState }) {
 
   const [filteredDocs, setFilteredDocs] = useState([...sortedDocs]);
 
-  const [selectedTags, setSelectedTags] = useState([]);
+  const queryTagsToLower = new Set(decodedQuery.split(/\s/)
```

Comment:
`queryTagsToLowerAndLongerThan3` :)) to be completely accurate.
Might I suggest `sanitizedQueryTags`?